frontend/src/RootLayout.tsx

```diff
@@ -13,3 +54,14 @@ const RootLayout = () => {
 }
 
 export default RootLayout
+
+export async function loader() {
+    const response = await fetch('http://localhost:8080/api/auth/loginStatus', {
+        credentials: 'include',
+    })
+    if (response.status !== 200) {
+        throw new Response(JSON.stringify({ message: 'Error fetching data' }))
+    }
```

Comment:
What does mean the line ?
```js
throw new Response(JSON.stringify({ message: 'Error fetching data' }))
```
* [mozilla: `throw`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw)
```suggestion
    if (response.status !== 200) {
        throw new Error('Error fetching data');
    }
```

Reply:
What happend when you use what you are proposing?
Have you tried your snippet?

Reply:
> What does mean the line ?
> 
> ```js
> throw new Response(JSON.stringify({ message: 'Error fetching data' }))
> ```
> 
>     * [mozilla: `throw`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw)

It sends a Response Object to ErrorPage

Reply:
Thanks for the resources